web/src/pm_list_data.ts

```diff
@@ -45,20 +45,20 @@ type DisplayObject = {
 };
 
 export function get_conversations(): DisplayObject[] {
-    const private_messages = pm_conversations.recent.get();
+    const direct_messages = pm_conversations.recent.get();
```

Comment:
This name is poor; I expect `conversations` would be a lot better, since these are NOT messages.